### PR TITLE
fixes accordion cards functionality

### LINKED ISSUE #1970: The accordion cards under https://distributed-system-analysis.github.io/pbench/start.html#install don't work.
Clicking on the `>` does not open them.

### PR DESCRIPTION
**Pull request by anishaswain**
fixes #1970

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -48,7 +48,7 @@ a{
 }
 
 ul{
-  list-style: inside;
+  list-style: none;
```
Comment (ndokos) [inline css/style.css:51]: I presume this is the fix. I don't understand it but it works!

**Review by ndokos — APPROVED** [review]
Works for me.

File: css/style.css (hunk 1 above)
Comment (dbutenhof) [reply css/style.css:51]: I was actually kinda wondering that, too; my guess that it was partly setting the click handlers on a page load event; but if the change in the `ul` style is responsible, I have no idea how.

**Review by dbutenhof — APPROVED** [review]

File: css/style.css (hunk 1 above)
Comment (anishaswain) [reply css/style.css:51]: @ndokos @dbutenhof I saw another design bug while fixing the accordion issue. The navbar was having list-style dots in them. So fixed them here. I thought of adding a comment but then felt adding a comment for only online in CSS might be inconsistent.

Comment (dbutenhof) [reply css/style.css:51]: Thanks, Anisha! That makes sense.

**Review by portante — APPROVED** [review]

**Pull request merged**